src/Backend/Modules/Pages/Engine/Model.php

```diff
@@ -1657,4 +1671,24 @@ private static function updateUrlAfterMove(int $pageId, array $page, int $newPar
 
         $database->update('meta', ['url' => $newUrl], 'id = ?', [$page['meta_id']]);
     }
+
+    private static function getImage(?string $image, string $metaUrl): ?string
```

Comment:
Niet logischer om dit `copyImage` te noemen? of `getCopiedImage`

Reply:
Fixed 686917856